strfry/plugins/policies.ts

```diff
@@ -53,6 +54,9 @@ const policies = [
       whitelist: [localhost, eventsIp],
     },
   ],
+
+  // Broadcast vanish requests to Redis
+  broadcastVanishRequests,
```

Comment:
It would be great if we could somehow write a test to verify that this is run _after_ signature verification.

Reply:
@mplorentz the verification happens in the strfry ingester, then the plugins are run before passing to the writer. I'm asking in the telegram channel just to be sure. But all we see in any plugin is already valid and their signatures are verified.

Doing a manual check to protect against any weird bug or future changes wouldn't hurt though but I'd be very surprised if that happens and it would be a major bug

Reply:
Ok, sounds like we don't need an automated test. It seemed possible to me that some folks might be running some spam filters _before_ signature verification as an optimization. But if that's not even possible in strfry with the plugin system then that's good enough for me 👍